policyengine_us/variables/gov/states/sc/tax/income/sc_taxable_income.py

```diff
@@ -8,3 +8,6 @@ class sc_taxable_income(Variable):
     defined_for = StateCode.SC
     unit = USD
     definition_period = YEAR
+
+    adds = ["sc_taxable_income_less_salt", "sc_additions"]
```

Comment:
do we have a `sc_taxable_income_less_salt` variable?

Reply:
or should it be `sc_fed_taxable_income`

Reply:
my bad, forgot to change the variable name, they meant to be the same thing. Which name is better: `sc_taxable_income_less_sal`t or `sc_fed_taxable_income`?

Reply:
I think we can use `sc_fed_taxable_income` to show that it is a duplicate of the federal variable

Reply:
I'd suggest `sc_federal_taxable_income_without_salt_deduction`